tests/rpc/calls/getTransactionReceipt.test.ts

```diff
@@ -0,0 +1,78 @@
+import { getTransactionReceiptHandler } from '../../../src/rpc/calls/getTransactionReceipt'
+import { RPCResponse } from '../../../src/types/types'
+
+describe('Test get transaction receipt request testnet', () => {
+  it('Returns transaction receipt', async () => {
```

Comment:
Can you also add failing tests, like non exist tx, wrong parameters etc.

Reply:
I've added failing tests for non-existent transactions and missing parameters.

https://github.com/berzanorg/rosettanet/commit/97d7f4f00b6f6b59b369474b7f1b25d0f58c6abd